gubernator/github/classifier.py

```diff
@@ -22,6 +22,7 @@
 
 
 XREF_RE = re.compile(r'k8s-gubernator.appspot.com/build(/[^])\s]+/\d+)')
+APPROVERS_RE = re.compile(r'<!-- META={"?approvers"?:\[([^]]*)\]} -->')
```

Comment:
Comment that this needs to match the munger's output, and maybe also comment around [here](https://github.com/kubernetes/test-infra/blob/master/mungegithub/mungers/approvers/owners.go#L476).